app/src/util/canvasPointRendering.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  getRadius: (weight, /* sigma */ zoom) => {
+    // seems like sigma is already zoom dependant?
+    // const normalizedSigma = Math.max(1, 3 * (Math.log(1+sigma)/zoomMult));
```

Comment:
ATM I used the value encoded in symbol size is weight, while it should be sigma. I have to investigate what this sigma value means and how it works - it seems that its value changes depending on the zoom level. Could that be possible? I pulled my hair while trying to implement zoom dependent styling because it seems so.